docs/book/src/cronjob-tutorial/testdata/project/config/manager/manager.yaml

```diff
@@ -44,12 +44,37 @@ spec:
       #             values:
       #               - amd64
       #               - arm64
-      #               - ppc64le
-      #               - s390x
       #           - key: kubernetes.io/os
       #             operator: In
       #             values:
       #               - linux
+      # TODO(user): Uncomment the following code to configure the nodeSelector expression
+      # annotations, nodeSelector, tolerations, topologySpreadConstraints.
+      # nodeSelector:
```

Comment:
Should it be really added there or via kustomize? Is there a better way to configure it?
We will need to check out and either validate the scenario with e2e tests. 
Could we have a PR for those and keep this one only for the imagePull ?